src/network/ThreeWindingsTransformerNetworkFactory.cpp

```diff
@@ -0,0 +1,246 @@
+#include <powsybl/iidm/LoadAdder.hpp>
+#include <powsybl/iidm/RatioTapChanger.hpp>
+#include <powsybl/iidm/RatioTapChangerAdder.hpp>
+#include <powsybl/iidm/Substation.hpp>
+#include <powsybl/iidm/SubstationAdder.hpp>
+#include <powsybl/iidm/ThreeWindingsTransformer.hpp>
+#include <powsybl/iidm/ThreeWindingsTransformerAdder.hpp>
+#include <powsybl/iidm/TopologyKind.hpp>
+#include <powsybl/iidm/VoltageLevel.hpp>
+#include <powsybl/iidm/VoltageLevelAdder.hpp>
+#include <powsybl/stdcxx/DateTime.hpp>
+
+
+namespace powsybl {
+
+namespace network {
+
+iidm::Network ThreeWindingsTransformerNetworkFactory::create() {
+    iidm::Network network("three-windings-transformer", "test");
+    network.setCaseDate(stdcxx::DateTime::parse("2018-03-05T13:30:30.486+01:00"));
+
+    iidm::Substation& substation = network.newSubstation()
+        .setId("SUBSTATION")
+        .setCountry(iidm::Country::FR)
+        .add();
+    iidm::VoltageLevel& vl1 = substation.newVoltageLevel()
+        .setId("VL_132")
+        .setNominalVoltage(132.0)
+        .setLowVoltageLimit(118.8)
+        .setHighVoltageLimit(145.2)
+        .setTopologyKind(iidm::TopologyKind::BUS_BREAKER)
+        .add();
+    iidm::Bus& bus132 = vl1.getBusBreakerView().newBus()
+        .setId("BUS_132")
+        .add();
+    bus132.setV(133.584).setAngle(-9.62);
+    vl1.newGenerator()
+        .setId("GEN_132")
+        .setBus("BUS_132")
+        .setMinP(0.0)
+        .setMaxP(140)
+        .setTargetP(7.2)
+        .setTargetV(135)
+        .setVoltageRegulatorOn(true)
+        .add();
+
+    iidm::VoltageLevel& vl2 = substation.newVoltageLevel()
+        .setId("VL_33")
+        .setNominalVoltage(33.0)
+        .setLowVoltageLimit(29.7)
+        .setHighVoltageLimit(36.3)
+        .setTopologyKind(iidm::TopologyKind::BUS_BREAKER)
+        .add();
+    iidm::Bus& bus33 = vl2.getBusBreakerView().newBus()
+        .setId("BUS_33")
+        .add();
+    bus33.setV(34.881).setAngle(-15.24);
+    iidm::Load& load33 = vl2.newLoad()
+        .setId("LOAD_33")
+        .setBus("BUS_33")
+        .setP0(11.2)
+        .setQ0(7.5)
+        .add();
+    load33.getTerminal()
+        .setP(11.2)
+        .setQ(7.5);
+
+    iidm::VoltageLevel& vl3 = substation.newVoltageLevel()
+        .setId("VL_11")
+        .setNominalVoltage(11.0)
+        .setLowVoltageLimit(9.9)
+        .setHighVoltageLimit(12.1)
+        .setTopologyKind(iidm::TopologyKind::BUS_BREAKER)
+        .add();
+    iidm::Bus& bus11 = vl3.getBusBreakerView().newBus()
+        .setId("BUS_11")
+        .add();
+    bus11.setV(11.781).setAngle(-15.24);
+    iidm::Load& load11 = vl3.newLoad()
+        .setId("LOAD_11")
+        .setBus("BUS_11")
+        .setP0(0.0)
+        .setQ0(-10.6)
+        .add();
+    load11.getTerminal()
+        .setP(0.0)
+        .setQ(-10.6);
+
+    iidm::ThreeWindingsTransformer& twt = substation.newThreeWindingsTransformer()
+        .setId("3WT")
+        // TODO(mathbagu) .setRatedU0(132.0)
```

Comment:
Why TODO ?

Reply:
This method doesn't exist yet. It appears in the new 3WT API (#48). This is a good catch. I have to think to fix it when this PR will be merged